### PR TITLE
Memedog from Undertale now orderable from cargo

### PR DESCRIPTION
**Pull request by Weisyl**
The annoying white little dog can now be ordered from cargo as a gimmick type pet thing. Also made wheelchairs move a little faster.
http://puu.sh/ljL4E/4c8f9f18e2.png

### CODE REVIEW

**Comment by McHatters** [comment]
don't forget the dog salad

**Comment by Chronitonity** [comment]
:+1:

File: code/modules/admin/verbs/modifyvariables.dm [hunk 1]
```diff
@@ -49,7 +49,7 @@ var/list/VVckey_edit = list("key", "ckey")
 	switch(class)
 
 		if("text")
-			var_value = input("Enter new text:","Text") as null|message
```
Comment (Carbonhello) [inline code/modules/admin/verbs/modifyvariables.dm:52]: ????

File: code/game/objects/structures/crates_lockers/closets/crittercrate.dm [hunk 2]
```diff
@@ -49,6 +49,10 @@
 		content_mob = /mob/living/simple_animal/pet/dog/corgi/Lisa
 	..()
 
+/obj/structure/closet/critter/memedog
+	name = "Strange Dog Crate"
```
Comment (Carbonhello) [inline code/game/objects/structures/crates_lockers/closets/crittercrate.dm:53]: Name should be "strange dog crate" for when you examine it imo

**Comment by danhibiki** [comment]
Why not?

File: code/modules/admin/verbs/modifyvariables.dm (hunk 1 above)
Comment (Weisyl) [inline code/modules/admin/verbs/modifyvariables.dm:52]: I dunno why the fuck this file was changed, I literally never even touched it.

**Comment by Lawligagger** [comment]
Alrighty

**Pull request merged**